documentation/staging/content/faq/newbie.md

```diff
@@ -119,10 +119,10 @@ For more information, see the [Domain life cycle]({{< relref "/userguide/managin
 
 **A:** Download pre-patched images from the Oracle Container Registry or use WebLogic Image Tool to create your own.
 For additional information on WebLogic container images on Oracle Container registry,
-see [Obtain images from the Oracle Container Registry]({{< relref "/userguide/base-images/_index.md#obtain-images-from-the-oracle-container-registry" >}}).
+see [Obtain images from the Oracle Container Registry]({{< relref "/userguide/base-images/ocr-images#obtain-images-from-the-oracle-container-registry" >}}).
 
 For additional information on creating new container images with the WebLogic Image Tool,
-see [Patch WebLogic images]({{< relref "/userguide/base-images/_index.md#create-a-custom-image-with-patches-applied" >}}).
+see [Patch WebLogic images]({{< relref "/userguide/base-images/ocr-images#create-a-custom-image-with-patches-applied" >}}).
```

Comment:
This location is now in 'custom-images', not 'ocr-images', but, since we are providing general guidance, '_index.md' without any '#' would probably be the best link

Reply:
I disagree. 

The first answer, "Download pre-patched images from the Oracle Container Registry or use WebLogic Image Tool to create your own. For additional information on WebLogic container images on Oracle Container registry, see [Obtain images from the Oracle Container Registry](http://localhost:1313/weblogic-kubernetes-operator/userguide/base-images/ocr-images/#obtain-images-from-the-oracle-container-registry)," points them to the OCR doc.

The next statement, "For additional information on **creating new container images with the WebLogic Image Tool** is best referenced with this doc, [Create custom images](http://localhost:1313/weblogic-kubernetes-operator/userguide/base-images/custom-images/).

Reply:
Specifically, there is no "create-a-custom-image-with-patches-applied" in the "ocr-images" doc, and the ocr-images doc is _not_ about creating images with WIT, which is _not_ the ocr-images doc...